src/core/ready.js

```diff
@@ -34,15 +34,6 @@ jQuery.extend( {
 	// the ready event fires. See #6781
```

Comment:
When removing this we may want to note in a comment that Migrate needs `readyWait` to shim `.holdReady()`. If/when .holdReady()` is removed in v4 we might be tempted to remove or simplify this logic, or make `readyWait` private. A similar thing happened in 3.0 when we removed `jQuery.sub()` and simplified `jQuery.fn.init()` so I'm just being paranoid.